lightning/src/util/events.rs

```diff
@@ -1011,6 +1011,23 @@ impl MaybeReadable for Event {
 				};
 				f()
 			},
+			25u8 => {
+				let f = || {
+					let mut prev_channel_id = [0; 32];
+					let mut failed_next_destination_opt = None;
+					read_tlv_fields!(reader, {
+						(0, prev_channel_id, required),
+						(2, failed_next_destination_opt, ignorable),
```

Comment:
I believe it's because `HTLCDestination::NextHopChannel` may fail to deserialize when serialized with a version prior to 0.0.110? And instead of handing the error up the chain and quitting deserialization altogether, we just don't read this event? If that's the case, a comment explaining this would indeed be appreciated.